api/v2/echo.go

```diff
@@ -196,6 +196,58 @@ func (s *server) RWMutexFind(c echo.Context) error {
 	return nil
 }
 
+func (s *server) ConcurrencySearchIndex(c echo.Context) error {
```

Comment:
This function is still using the same method as the SearchValue
and if you search the index it will still search for the value
for example the following requests are the same
```bash
curl -XGET localhost:8080/v2/numbers/concurrency/index/99
{"index":85,"value":99}
curl -XGET localhost:8080/v2/numbers/concurrency/value/99
{"index":85,"value":99}

```